sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedColumnReader.java

```diff
@@ -894,4 +904,15 @@ private void readPageV2(DataPageV2 page) throws IOException {
       throw new IOException("could not read page " + page + " in col " + descriptor, e);
     }
   }
+
+  private boolean isTimestampTypeMatched(TimeUnit unit) {
```

Comment:
nit: these can be static methods

Reply:
Should we extract these methods into util class?

Reply:
I mean we can just change `private boolean` to `private static boolean` since they are not depending on any of the state from the class, but it's a really minor thing :)